pkg/api/v1beta3/defaults.go

```diff
@@ -25,6 +25,21 @@ import (
 
 func init() {
 	api.Scheme.AddDefaultingFuncs(
+		func(obj *ReplicationController) {
+			var labels map[string]string
+			if obj.Spec.Template != nil {
+				labels = obj.Spec.Template.Labels
+			}
+			// TODO: support templates defined elsewhere when we support them in the API
```

Comment:
Yeah, multi-object lookups don't work from this code at all, so that'll be interesting when it becomes a thing.